app/script/components/list/participantItem.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+'use strict';
+
+window.z = window.z || {};
+window.z.components = z.components || {};
+
+z.components.ParticipantItem = class ParticipantItem {
+  constructor(params) {
+    this.participant = ko.unwrap(params.participant);
+    this.isService = this.participant instanceof z.integration.ServiceEntity || this.participant.isBot;
+    this.isUser = this.participant instanceof z.entity.User && !this.participant.isBot;
+    this.selfUser = window.wire.app.repository.user.self;
+    this.contentInfo = '';
```

Comment:
`this.contentInfo = this.isService ? this.participant.summary : this.participant.username()` and set the default value of username in L134 of the User entity to '';